src/OrchardCore.Cms.Web/appsettings.json

```diff
@@ -32,6 +32,43 @@
     //  "BlobName": "", // Optional, defaults to Sites/tenant_name/DataProtectionKeys.xml. Templatable, refer docs.
     //  "CreateContainer": true // Creates the container during app startup if it does not already exist.
     //},
+    "OrchardCore_Email": {
+      "DefaultSender": "",
+      // For backward compatibility, the following settings are used if the corresponding settings in the "OrchardCore_Email_Smtp" section are not set.
+      "DeliveryMethod": "Network",
+      "PickupDirectoryLocation": "",
+      "Host": "localhost",
+      "Port": 25,
+      // Uncomment if SMTP server runs through a proxy server
+      //"ProxyHost": "proxy.domain.com",
+      //"ProxyPort": 5050,
+      "EncryptionMethod": "SSLTLS",
+      "AutoSelectEncryption": false,
+      "UseDefaultCredentials": false,
+      "RequireCredentials": true,
+      "Username": "",
+      "Password": ""
```

Comment:
We don't really need the backward compatibility demonstrated like this here, because people might mistakenly copy-paste this instead of the current one.